client/src/pages/Dashboard/TestUpload.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import PhotoCamera from '@material-ui/icons/PhotoCamera';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '& > *': {
+        margin: theme.spacing(1),
+      },
+    },
+    input: {
+      display: 'none',
+    },
+  }),
+);
+
+export default function UploadButtons() {
+  const classes = useStyles();
+  const [img, setImg] = useState<any>(null);
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const fetchOptions = {
+    method: 'POST',
+    image: img,
+  };
+  useEffect(() => {
+    fetch(`/profile`, fetchOptions).then((res) => res.json());
+    console.log('useEffect fired');
```

Comment:
Remove console log

Reply:
I deleted the whole file since in this ticket I have to set up only the route, not the front-end.